Using Hocon substitutions creates ConfigException$NotResolved error

When using Hocon substitutions in a DataObject, then trying to get suggestions for inputId of a CopyAction, the following error happens in the log:
```
Caused by: com.typesafe.config.ConfigException$NotResolved: need to Config#resolve(), see the API docs for Config#resolve(); substitution not resolved: ConfigReference(${env.tablePathWithId})
	at com.typesafe.config.impl.ConfigReference.notResolved(ConfigReference.java:35)
	at com.typesafe.config.impl.ConfigReference.unwrapped(ConfigReference.java:47)
	at com.typesafe.config.impl.SimpleConfigObject.unwrapped(SimpleConfigObject.java:243)
	at com.typesafe.config.impl.SimpleConfigObject.unwrapped(SimpleConfigObject.java:26)
	at com.typesafe.config.impl.SimpleConfigObject.unwrapped(SimpleConfigObject.java:243)
	at io.smartdatalake.context.ContextAdvisorImpl.retrieveDataObjectIds(ContextAdvisorImpl.scala:21)
	at io.smartdatalake.context.ContextAdvisorImpl.generateSuggestions(ContextAdvisorImpl.scala:15)
	at io.smartdatalake.completion.SDLBCompletionEngineImpl.generateCompletionItems(SDLBCompletionEngineImpl.scala:22)
	at io.smartdatalake.languageserver.SmartDataLakeTextDocumentService.completion$$anonfun$1(SmartDataLakeTextDocumentService.scala:55)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:687)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
```

To reproduce (all files in the same directory, workspaceType = ActiveWorkspace, workspaceParameters = "dir"):

dev.conf
```
env {
  catalog = test
}
```

dataObjects.conf
```
dataObjects {
  csvTest {
    type = CsvFileDataObject
    path = "test"
    csvOptions {
      delimiter = ","
    }
  }
  slvTest {
    type = DeltaLakeTableDataObject
    table {
      catalog = ${env.catalog}
      db = "slv"
      name = "test"
    }
  }
}
```

actions.conf
```
actions {
  copyCsv {
    type = CopyAction
    inputId = csvTest
    outputId = 
  }
}
```

Then use lsp suggestion on `actions.copyCsv.outputId`

--- a/src/main/scala/io/smartdatalake/context/TextContext.scala
+++ b/src/main/scala/io/smartdatalake/context/TextContext.scala
@@ -23,7 +23,7 @@ case class TextContext private (uri: String, originalText: String, workspaceUriT
     if newConfig == HoconParser.EMPTY_CONFIG then
       copy(originalText=newText, isConfigCompleted=isConfigCompleted)
     else
-      copy(originalText=newText, configText=newConfigText, rootConfig=newConfig, isConfigCompleted=isConfigCompleted)
+      copy(originalText=newText, configText=newConfigText, rootConfig=newConfig.resolve(), isConfigCompleted=isConfigCompleted)
 
   override def toString: String = s"TextContext(originalText=${originalText.take(50)}, configText=${configText.take(50)}, rootConfig=${rootConfig.toString})"
 
@@ -35,4 +35,4 @@ object TextContext:
     val configText = MultiLineTransformer.flattenMultiLines(originalText)
     val fullText = (configText::workspaceUriToContents.removed(uri).values.toList).mkString("\n")
     val config = HoconParser.parse(fullText).getOrElse(HoconParser.EMPTY_CONFIG)
-    TextContext(uri, originalText, workspaceUriToContents, configText, config)
+    TextContext(uri, originalText, workspaceUriToContents, configText, config.resolve())

--- a/src/main/scala/io/smartdatalake/schema/SchemaReaderImpl.scala
+++ b/src/main/scala/io/smartdatalake/schema/SchemaReaderImpl.scala
@@ -105,5 +105,4 @@ class SchemaReaderImpl(val schemaPath: String) extends SchemaReader with SDLBLog
       case _ => None)
     case _ => None
 
-
 }

